src/components/Homepage/LendByCategory/LoanCategory.vue

```diff
@@ -72,12 +55,13 @@
 					</div>
 				</div>
 			</div>
-			<span
+			<button
 				class="arrow right-arrow"
```

Comment:
I've got the same problem with carousel right now, but I think we need a new chevron added to KvIcon's sprite. I'll add that hopefully tomorrow. We'll need to get our button styling between this carousel and the one to come above too.

Reply:
👍  Im not super invested in these, we can make the chevron in a circle it's own mini component